hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneKeyDetails.java

```diff
@@ -95,4 +102,13 @@ public FileEncryptionInfo getFileEncryptionInfo() {
   public void setOzoneKeyLocations(List<OzoneKeyLocation> ozoneKeyLocations) {
     this.ozoneKeyLocations = ozoneKeyLocations;
   }
+
+  /**
+   * Get OzoneInputStream to read the content of the key.
+   * @return OzoneInputStream
+   * @throws IOException
+   */
+  public OzoneInputStream getContent() throws IOException {
```

Comment:
`OzoneKeyDetails` is converted to JSON by `ozone sh key info`.  The `getContent()` method should be excluded from the conversion, because we don't want the binary content to be shown along with metadata.

```suggestion
  @JsonIgnore
  public OzoneInputStream getContent() throws IOException {
```

This prevents:

```
No serializer found for ... org.apache.hadoop.ozone.client.OzoneKeyDetails["content"]->org.apache.hadoop.ozone.client.io.OzoneInputStream["inputStream"]->...
```

https://github.com/apache/ozone/runs/7554046528?check_suite_focus=true#step:5:253